apps/files/src/mixins.js

```diff
@@ -447,9 +447,22 @@ export default {
         // FIXME: this might break if relativePath is not the currentFolder
         // and is a mount point that has no chunk support
         if (this.browserSupportsChunked && this.currentFolder.isChunkedUploadSupported) {
+          let chunkSize = this.configuration.uploadChunkSize
+          if (this.capabilities.files.tus_support.max_chunk_size > 0) {
+            if (
+              chunkSize === null ||
+              chunkSize === 0 ||
+              chunkSize > this.capabilities.files.tus_support.max_chunk_size
+            ) {
+              chunkSize = this.capabilities.files.tus_support.max_chunk_size
+            }
+          }
+          console.log('tus_support: ', this.capabilities.files.tus_support)
+          console.log('Using chunk size: ', chunkSize)
```

Comment:
Forgotten as well?

Reply:
yeah... used the undo feature of my editor and didn't see that it brought those back as well :facepalm: